.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
 env:
   # Go version we currently use to build containerd across all CI.
   # Note: don't forget to update `Binaries` step, as it contains the matrix of all supported Go versions.
-  GO_VERSION: "1.20.13"
+  GO_VERSION: "1.21.8"
```

Comment:
I think we missed one change in https://github.com/containerd/containerd/pull/9890. Because after setting up composite action, we should not be  setting `GO_VERSION` env in the workflow file.

Reply:
Included the missing change in the action now, but this file doesn't seem to be updated to use the action

Reply:
Have pushed the changes here https://github.com/dmcgowan/containerd/pull/3